src/sdk/userDecrypt.ts

```diff
@@ -70,20 +70,20 @@ export const reencryptRequest =
     let response;
     let json;
     try {
-      response = await fetch(`${gatewayUrl}reencrypt`, options);
+      response = await fetch(`${relayerUrl}reencrypt`, options);
```

Comment:
the url itself is not renamed ?

Reply:
like user/private decrypt

Reply:
I don't know at this moment. I just renamed gateway to relayer